src/tmpfiles/tmpfiles.c

```diff
@@ -2261,12 +2310,64 @@ static int remove_item(Item *i) {
         }
 }
 
+static int age_by_format(AgeBy age_by, char **ret_age_by_fmt) {
+        _cleanup_free_ char *tmp = NULL;
+        _cleanup_strv_free_ char **split = NULL;
+        char types[5] = { 0 };
+        char *p = NULL, *q = NULL;
+
+        assert(ret_age_by_fmt);
+
+        if (age_by == AGE_BY_UNSET) {
+                *ret_age_by_fmt = strdup("");
+                if (!*ret_age_by_fmt)
```

Comment:
see https://systemd.io/CODING_STYLE/

Please do not clobber return params on failure. we have the strict rule that return params should not be touched on failure, and unconditionally initialized on success. But here, if the strdup() fails you'll have reset the `ret_age_by_fmt` parameter. hence, please initialize this into temp variable first, and move it over then.